io/src/main/scala/sbt/io/RichURI.scala

```diff
@@ -50,7 +50,9 @@ class RichURI(uri: URI) {
     }
 }
 
-object RichURI {
+object RichURI extends URIExtra
+
+trait URIExtra {
```

Comment:
I'm not sure if that's a *good* style to mimic or not.   I'd hate to expose a public trait without a good reason first.